documentation/docs/how_tos/post_installation.md

```diff
@@ -13,151 +13,34 @@ keywords:
+Please see the [`config.json`](../references/configuration.md) and [`peering.json`](../references/peering.md) articles for more information about the contents of the configuration files.
 
 ## Configuring HTTP REST API
 
-One of the tasks the the node is responsible for is exposing the [HTTP REST ](../explanations/nodes_101.md#http-rest-api) to clients that would like to interact with the IOTA network, such as crypto wallets, exchanges, IoT devices, etc.
+One of the tasks the the node is responsible for is exposing [API](../references/api_reference.md) to clients that would like to interact with the IOTA network, such as crypto wallets, exchanges, IoT devices, etc.
 
-By default, Hornet will publicly expose the [HTTP REST ](../explanations/nodes_101.md#http-rest-api) on port 14265, as it is ready to accept incoming connections from the Internet.
+By default, HORNET will expose the [Core REST API v2](../references/api_reference.md) on port `14265`. If you use the [recommended setup](using_docker.md) the API will be exposed on the default HTTPS port (`443`) and secured using an SSL certificate.
 
 Since offering the HTTP REST API to the public can consume your node's resources, there are options to restrict which routes can be called and other request limitations.
 
-You can find the HTTP REST API related options in the `restAPI` section within the `config.json` file:
-
-```json
-  "restAPI": {
-    "jwtAuth": {
-      "salt": "HORNET"
-    },
-    "publicRoutes": [
-      "/health",
-      "/api/routes",
-      "/api/core/v2/info",
-      "/api/core/v2/tips",
-      "/api/core/v2/blocks*",
-      "/api/core/v2/transactions*",
-      "/api/core/v2/milestones*",
-      "/api/core/v2/outputs*",
-      "/api/core/v2/treasury",
-      "/api/core/v2/receipts*"
-    ],
-    "protectedRoutes": [
-      "/api/*",
-    ],
-    "bindAddress": "0.0.0.0:14265",
-    "pow": {
-      "enabled": true,
-      "workerCount": 1
-    },
-    "limits": {
-      "bodyLength": "1M",
-      "maxResults": 1000
-    }
-  }
-```
-
-If you want to make the HTTP REST API only accessible from localhost, you change the `restAPI.bindAddress` config option accordingly.
-
-`restAPI.publicRoutes` defines which routes can be called without JWT authorization. `restAPI.protectedRoutes` defines which routes require JWT authorization. All other routes will not be exposed.
-
-If you are concerned with resource consumption, consider turning off `restAPI.pow.enabled`. This way, the clients must perform proof of work locally before submitting a block for broadcast. If you would like to offer proof of work to clients, consider increasing the `restAPI.pow.workerCount` to provide a faster block submission experience.
-
-We recommend that you provide your HTTP REST API behind a reverse proxy, such as [HAProxy](http://www.haproxy.org/), [Traefik](https://traefik.io/), [Nginx](https://www.nginx.com/), or [Apache](https://www.apache.org/) configured with TLS.
```

Comment:
Do we really want to remove that suggestion?
Maybe we can add that this is done automatically if they use the docker setup.